packages/atomic/src/components/searchbox/base-search.tsx

```diff
@@ -0,0 +1,182 @@
+import {Component, Event, EventEmitter, h, Prop} from '@stencil/core';
+import {I18nState} from '../../utils/initialization-utils';
+import {Combobox} from '../../utils/combobox';
+
+import ClearIcon from 'coveo-styleguide/resources/icons/svg/clear.svg';
+import SearchIcon from 'coveo-styleguide/resources/icons/svg/search.svg';
+
+@Component({
+  tag: 'base-search',
+  styleUrl: 'base-search.pcss',
+  shadow: false,
```

Comment:
Since it's an internal component, it will always be encapsulated into the shadow dom of the component that contains it. We don't really want or need nested shadow doms (this will just make it harder for us to style internal components).